source/renderer/app/components/staking/rewards/StakingRewardsForIncentivizedTestnet.js

```diff
@@ -307,13 +355,19 @@ export default class StakingRewardsForIncentivizedTestnet extends Component<
                         reward,
                         REWARD_FIELDS.REWARD
                       ).toFormat(DECIMAL_PLACES_IN_ADA);
+                      const rewardsAddress = get(
+                        reward,
+                        REWARD_FIELDS.REWARDS_ADDRESS
+                      );
 
                       return (
                         <tr key={key}>
                           <td>{rewardWallet}</td>
-                          <td>
+                          <td className={styles.rewardAmount}>
                             {isRestoring ? '-' : rewardAmount}
-                            {isRestoring && (
+                          </td>
+                          <td className={styles.rewardsAddress}>
+                            {isRestoring ? (
```

Comment:
@aleksandardjordjeviciohk please make sure we show the rewards address during restoration too.
cc @ManusMcCole